sources/packages/web/src/services/ApplicationOfferingChangeRequestService.ts

```diff
@@ -197,4 +198,19 @@ export class ApplicationOfferingChangeRequestService {
       payload,
     );
   }
+
+  /**
+   * Approves or declines the application offering change request status.
+   * @param applicationOfferingChangeRequestId application offering change request id.
+   * @param payload information to update the application offering change request.
+   */
+  async updateAESTApplicationOfferingChangeRequestStatus(
```

Comment:
The name `updateAESTApplicationOfferingChangeRequestStatus` doesn't seem to be appropriate. when ministry approves we do more than updating status. I would suggest `AssessApplicationOfferingChangeRequest`